vcd/resource_vcd_vapp_vm.go

```diff
@@ -1500,6 +1604,9 @@ func networksToConfig(networks []interface{}, vdc *govcd.Vdc, vapp govcd.VApp, v
 		netConn := &types.NetworkConnection{}
 
 		networkName := nic["name"].(string)
+		if networkName == "" {
```

Comment:
Is there a specific reason for it? Looks to have worked until now.

Reply:
yes, there is. My API part I use, requires this explicitly. Existing API don't care